index.js

```diff
@@ -525,16 +526,21 @@ class ResembleHelper extends Helper {
       throw new Error('Cannot get element size!');
     }
 
-    const bottom = size.height + location.y;
-    const right = size.width + location.x;
+    const scrollbarCoordinates = await browser.execute('return { offsetX: window.pageXOffset, offsetY: window.pageYOffset }');
```

Comment:
I would use little bit different names:
```
const scrollOffset = await browser.execute('return { X: window.pageXOffset, Y: window.pageYOffset }');
```
So then the usage will look like this:
```
scrollOffset.X
```